### PR TITLE
Fix the logic to send webpa sync Notification for empty string value.

### PR DESCRIPTION
**Pull request by rajnishdey**

### CODE REVIEW

**Comment by rajnishdey** [comment]
Fix the logic to send webpa sync Notification for the empty string value.

File: source/broadband/cosa_webpa_dml.c [hunk 1]
```diff
@@ -90,8 +92,9 @@ Webpa_SetParamStringValue
                 {
                         WalError("Received insufficient data to process notification\n");
                 }
-
+                WAL_FREE(tmpStr);
         #endif
+                WAL_FREE(tmpStr);
```
Comment (shilpa24balaji) [inline source/broadband/cosa_webpa_dml.c]: Is this needed?

Comment (rajnishdey) [reply source/broadband/cosa_webpa_dml.c]: No, fixed in the latest patch.

**Review by shilpa24balaji — APPROVED** [review]

**Comment by shilpa24balaji** [comment]
@guruchandru Please merge this

**Pull request merged**